src/GraphQLBackend.ts

```diff
@@ -107,18 +108,12 @@ export class GraphQLBackendCreator {
         logger.info("Schema generation skipped.")
       }
 
-      backend.resolvers = {}
-      backend.resolvers["Query"] = {}
-      backend.resolvers["Mutation"] = {}
-      for (const value of this.resolvers) {
-        if(value.resolverType === 'Query') {
-          backend.resolvers["Query"][value.fieldName] = value.implementation
-        }
-        if(value.resolverType === 'Mutation') {
-          backend.resolvers["Mutation"][value.fieldName] = value.implementation
-        }
-      }
-
+      /**
+       * Generate TS resolvers from the ResolverInstance[] generated
+       */
+      const generator = new GraphQLResolverGenerator()
```

Comment:
Generator should be injected. Hardcoding genrator is not best option. 
What will happen if we will want to add another generator in future